modules/create-resource-central-db-bundle.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
```

Comment:
Ignore this file, I'll delete this. Not required anymore.

Reply:
Why did you plan to delete this? It looks useful to me. Any replacement for this?

Reply:
Sorry, I commented on the wrong PR on a wrong file.